CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# v1.15.1 (2023-10-9)
+
+## OS Changes
+
+* Removed explicit type for mounting filesystems to allow older ext4 snapshot volumes to be mounted in newer variants that default to xfs ([#3499])
+* Update to `apiclient` Rust dependencies ([#3491])
```

Comment:
```suggestion
* Update `apiclient` Rust dependencies ([#3491])
```
Or 

```suggestion
* Update tugnstenite for `apiclient` ([#3491])
```

Since we didn't actually updated all the dependencies.